Makefile

```diff
@@ -44,13 +24,35 @@ fmt:
 	@gofmt -s -w main.go
 	@go fmt ./...
 
+lint: fmt
```

Comment:
linting should not be running with fmt, during the build it will fmt before running the lint so builds may pass but the code may not be linted

Reply:
good catch

Reply:
done